go/api/config.go

```diff
@@ -276,8 +276,15 @@ func (config *GlideClientConfiguration) WithReadFrom(readFrom ReadFrom) *GlideCl
 // encompasses sending the request, awaiting for a response from the server, and any required reconnections or retries. If the
 // specified timeout is exceeded for a pending request, it will result in a timeout error. If not set, a default value will be
 // used.
+//
+// WARNING: If requestTimeout exceeds the max duration of 2^32 - 1, a default value will be used.
```

Comment:
This comment does not specify a unit, making it unclear what the maximum allowed `time.Duration` value is.